packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception/index.tsx

```diff
@@ -129,9 +134,7 @@ const Exception = ({ exception, discovery }: Props) => {
                 <Container
                     title="TR_AUTH_CONFIRM_FAILED_TITLE"
                     description="TR_AUTH_CONFIRM_FAILED_DESC"
-                    cta={{
-                        action: () => dispatch(suiteActions.authConfirm()),
-                    }}
+                    cta={{ action: suiteActions.authConfirm }}
```

Comment:
```suggestion
                    cta={{ action: actions.authConfirm }}
```
e2e is failing on this @matejkriz